config.toml

```diff
@@ -77,6 +83,7 @@ lastmod = ["lastmod", ":git", "date"]
 
 [[menu.main]]
   url = "#contact"
+
```

Comment:
This whitespace seems unnecessary here and most likely was leftover from your custom config I assume.

Reply:
oups deleted that on the new commit